### PR TITLE
Create AddConcerts page

### LINKED ISSUE #30: Add Concerts page
page for #24

### PR DESCRIPTION
**Pull request by Alkawi**
closes #30 

I extracted the Form component from Dashboard to its own page.

### CODE REVIEW

**Review by PReichetanz — APPROVED** [review]
Looks good to me!

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+import AddConcerts from './Pages/AddConcerts'
 import Dashboard from './Pages/Dashboard'
 
 function App(): JSX.Element {
-  return <Dashboard />
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Dashboard />} />
+        <Route path="/addConcert" element={<AddConcerts />} />
```
Comment (marwinburesch) [inline src/app/App.tsx]: You should ommit the `/` for `path`'s value
https://reactrouter.com/docs/en/v6/api#routes-and-route

File: src/app/Pages/AddConcerts.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { v4 as uuid } from 'uuid'
+import useLocalStorage from '../hooks/useLocalStorage'
+import { useNavigate } from 'react-router'
+
+export default function AddConcerts(): JSX.Element {
+  const navigate = useNavigate()
+  const [concerts, setConcerts] = useLocalStorage<Concert[] | null>(
+    'concerts',
+    null
+  )
```
Comment (marwinburesch) [inline src/app/Pages/AddConcerts.tsx:13]: This would look cooler if `useLocalStorage`'s initial value would be optional :)

**Review by marwinburesch — COMMENTED** [review]
Awesome work

**Pull request merged**